modules/programs/fish.nix

```
@@ -349,6 +349,18 @@ in {
               config.home.sessionVariables)
           }
 
+          for p in "$HOME/.nix-profile/bin"
+            if not contains $p $PATH
+              set PATH $p $PATH
```

```suggestion
              set -p PATH $p
```

Why prepend, though, and not append? (`set -a`)